app/src/main/java/ca/llamabagel/transpo/di/SearchModule.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2019 Derek Ellis. Subject to the MIT license.
+ */
+
+package ca.llamabagel.transpo.di
+
+import ca.llamabagel.transpo.data.SearchRepository
+import ca.llamabagel.transpo.data.db.TransitDatabase
+import dagger.Module
+import dagger.Provides
+
+@Module
+abstract class SearchModule {
+    @Module
+    companion object {
+        @Provides
+        @JvmStatic
+        fun provideSearchRepository(database: TransitDatabase) = SearchRepository(database)
```

Comment:
This provider (and the entire module really) isn't needed since Dagger can already provide a `SearchRepository` because its constructor is marked with `@Inject`.